specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-10-01/mfe.json

```diff
@@ -10990,6 +11215,72 @@
       "x-ms-discriminator-value": "custom_model",
       "additionalProperties": false
     },
+    "CustomMonitoringSignal": {
+      "required": [
+        "componentId",
+        "metricThresholds"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/MonitoringSignalBase"
+        }
+      ],
+      "properties": {
+        "componentId": {
+          "description": "[Required] ARM resource ID of the component resource used to calculate the custom metrics.",
+          "minLength": 1,
+          "pattern": "[a-zA-Z0-9_]",
+          "type": "string",
```

Comment:
Use [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details) to indicate an ARM resource.

Reply:
The comment was off - this is actually a reference to a component asset. Thank you!